sycl/include/sycl/accessor.hpp

```diff
@@ -2065,14 +2065,10 @@ class __SYCL_EBO __SYCL_SPECIAL_CLASS __SYCL_TYPE(accessor) accessor :
   }
 
   template <access::target AccessTarget_ = AccessTarget,
-            typename = detail::enable_if_t<AccessTarget_ ==
-                                           access::target::host_buffer>>
-#if SYCL_LANGUAGE_VERSION >= 202001
-  std::add_pointer_t<value_type> get_pointer() const noexcept
-#else
-  DataT *get_pointer() const
-#endif
```

Comment:
Since `-sycl-std=1.2.1` is still around, I would prefer we keep this guard around.

Reply:
thanks @steffenlarsen , done.